lib/shoryuken/environment_loader.rb

```diff
@@ -92,6 +92,7 @@ def load_rails
         require File.expand_path('config/environment.rb')
       end
 
+      Shoryuken.options.merge!(config_file_options)
```

Comment:
@laCour is it a possible fix for what @waynerobinson commented out [here](https://github.com/phstc/shoryuken/pull/191/files#r59998676)?